.claude/commands/add-enhancement.md

```diff
@@ -0,0 +1,80 @@
+---
+description: Create a new OpenShift Enhancement Proposal
+args:
+  - name: area
+    description: Enhancement area (subdirectory under enhancements/)
+  - name: name
+    description: One-line title describing the enhancement
+  - name: description
+    description: Detailed description (what, why, who)
+  - name: jira
+    description: JIRA ticket URL for tracking
+---
+
+You are tasked with creating a new OpenShift Enhancement Proposal based on the template at `guidelines/enhancement_template.md`.
```

Comment:
Perhaps make it explicit, adding: `Mirror all required headings from guidelines/enhancement_template.md exactly`